### PR TITLE
Improves combat defibrillator / defib balance change

### PR DESCRIPTION
**Pull request by ghost**
It now uses the belt slot, has an on-mob sprite, is smaller, and is red.

Also defibs can go through armor and armor children now. They still can't go through space suits but it got really annoying especially since ever sec officer wears armor.

### CODE REVIEW

File: code/game/objects/items/weapons/defib.dm [hunk 1]
```diff
@@ -345,7 +354,7 @@
 				var/total_brute	= 0
 				if(do_after(user, 20)) //placed on chest and short delay to shock for dramatic effect, revive time is 5sec total
 					for(var/obj/item/carried_item in H.contents)
-						if((istype(carried_item, /obj/item/clothing/suit/armor)) || (istype(carried_item, /obj/item/clothing/suit/space)))
+						if(istype(carried_item, /obj/item/clothing/suit/space))
```
Comment (Deantwo) [inline code/game/objects/items/weapons/defib.dm]: And this changes what?

Comment (pudl) [inline code/game/objects/items/weapons/defib.dm]: it makes it not shitcode

Comment (paprka) [inline code/game/objects/items/weapons/defib.dm]: oh sorry let me state that in the OP i forgot bout it

Comment (Deantwo) [inline code/game/objects/items/weapons/defib.dm]: Thank you.

**Comment by Steelpoint** [comment]
I think this is good. 

I was going to rename the ERT's Combat Defib to the Compact Defib, however I think differentiating the Defib used by the ERT, and maybe giving the CMO a station bound Compact Defib might be interesting.

Also allows for us to balance the ERT's Defib without impacting with the station's defib's, if we decide to give compact defibs to the station.

**Comment by paprka** [comment]
perhaps, then again there's really no difference. they both function the same. i could make the combat defib stupid OP and go through space suits though, for reviving fellow ERT members

**Comment by Steelpoint** [comment]
I was also thinking of having the combat defib inject some basic drugs into the patient, just something to stop oxyloss for a little bit.

Either way, just being able to use it in your belt slot is a big buff, but the more the better.

**Comment by paprka** [comment]
Gonna redo this

**Pull request closed without merging**